src/base-components/card-base.js

```diff
@@ -1,27 +1,20 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import clsx from 'clsx'
 
 import { Close } from '../icons'
+import { Layout } from '../index'
```

Comment:
it may seem unnatural at first, but you can just do:

```js
import { Layout } from '..'
```

however, this may also mean that it's probably better to directly associate to the `Layout` module instead of proxy through the main index, depending on how frequently these get changed:

```js
import Layout from '../components/layout'
```